rules.d/60-sensor.rules

```diff
@@ -3,6 +3,10 @@
 ACTION=="remove", GOTO="sensor_end"
 
 # device matching the sensor's name and the machine's DMI data for IIO devices
+SUBSYSTEM=="iio", KERNEL=="iio*", SUBSYSTEMS=="usb|i2c|platform", \
+  KERNELS=="HID-SENSOR-200073*", ATTR{[dmi/id]chassis_type}=="30|31|32", \
```

Comment:
This makes sense to me.

Reply:
HID-SENSOR-* is a platform driver created by ISH,  so please remove test on usb and i2c.

Reply:
Thanks for reviewing it, done.

Reply:
LGTM.